editor.planx.uk/src/routes/views/standalone.tsx

```diff
@@ -25,16 +25,19 @@ const standaloneView = async (req: NaviRequest) => {
   const data = await fetchDataForStandaloneView(flowSlug, teamSlug);
 
   const {
-    flows: [{ team, settings: flowSettings }],
+    flows: [{ team, flowSettings }],
     globalSettings,
   } = data;
 
+  useStore.setState({ flowSettings });
+
   const state = useStore.getState();
   state.setFlowNameFromSlug(flowSlug);
   state.setGlobalSettings(globalSettings[0]);
-  state.setFlowSettings(flowSettings);
   state.setTeam(team);
 
+  console.log('Standalone state: ', state)
```

Comment:
I've been trying to test the refactoring locally by logging out the state to ensure that the state is being set successfully in the relevant components. 

Although I'm struggling to figure out how to setup a `standalone` view 🤔 

I believe an example would be an invite to pay and I maybe just need to persevere to setup that up locally?

Reply:
This might add some context here - https://trello.com/c/0Uw8ViZJ/2324-make-editor-sidebar-preview-behave-same-as-the-unpublished-preview-increase-granularity-of-previewenvironment

It's a bit of a pain point (at least for me!) and something I'd love to have a crack at refactoring / removing / rationalising.

This is referring to the `PreviewEnvironment` variable in `shared.ts` (the SharedStore slice) - `standalone` is essentially all pages that are not the Editor 👌

Reply:
Thanks for the info on this @DafyddLlyr!

I'm still a bit confused because based on this I thought I'd see the output from `console.log('Standalone state: ', state)` when I was on a `/preview` route? 

Although that seems to use the `published` view rather than the `standalone` view?